src/rye-bread-loaf.md

```diff
@@ -0,0 +1,36 @@
+# Rye Bread Loaf
+
+![Rye Bread Loaf](pix/rye-bread-loaf.webp)
+
+Rye bread is the national food of Finland. This bread is both traditional as well as delicious and serves great variation to the basic wheat breads. Make sure to prepare yourself by making rye based sourdough starter 4-5 days before baking and start the preparations 1 day beforehand.
+
+- 🍽️ Servings: 3 medium sized loafs
+
+## Ingredients
+
+- 750g warm water (around 35C) 
+- 60g rye sourdough starter
+- 860g rye flour
+- 20g salt
+- Some yeast if you have low faith in your starter's activity (optional)
+
+## Directions
+
+1. First make 'raski' by combining 250g of water, 60g of rye sourdough starter and 200g of rye flour. Stir into even gruel.
+2. Cover the bowl with cloth and let it sit in room temperature for 12 hours.
+3. Take 60g of the 'raski' and add it back to your rye sourdough starter. This way you don't need to separately feed it more. Let it rest for another 12 hours. Rye bread gets its signature sour taste from the sourdough starter and slow fermentation. The colder the water and evironment is the sourer the bread is. Warmer water and environment will result in milder taste.
+4. Now the actual dough: add 500g of warm water (around 35C), 660g of rye flour, 20g of salt and optionally a small amount of yeast.
+5. Mix all the ingredients to even dough. You get the best results doing this by hand. You don't need to knead rye dough, it is enough to just mix the ingredients. The dough should be moist and sticky. Don't add more flour or the bread will be dry and hard. Don't be alarmed if you have made lots of wheat bread and are used to the dough being firm. 
+6. Let the dough rest in room temperature covered with a cloth for 1-2 hours. The dough should rise noticeably. It it hasn't either the temperature is too low or the rye sourdough starter isn't active enough. If this happens you can add some yeast to the dough and mix it again well. Let rise again for another hour.
+7. Divide the dough to three parts of two if you prefer larger breads. Make sure the table has plenty rye flour on it. The less flour you use the better the bread will be but as the dough is very sticky it is ok to use a lot more flour than with wheat bread
+8. With the least possible amount of handling form the dough pieces to round or elongated loafs. Easiest way is to turn the dough with your hand against the table making it as thick as possible as the dough will flatten in the oven.
+9. Sprinkle rye flour on top of the loafs and cover them with cloth and let rise for around 2-4 hours. The loafs have risen enough when noticable cracks have formed on them. Use this as an indication rather than following the clock too much.
+10. Heat the oven to 250C. Put the loafs in and lower the temperature to 230C. Bake them for 35-50 minutes until they have taken dark browm color evenly.
```

Comment:
```suggestion
10. Heat the oven to 250°C. Put the loafs in and lower the temperature to 230°C. Bake them for 35-50 minutes until they have taken dark brown color evenly.
```
I don't know much about cooking, are you sure it's 250°C, that's more than twice the boiling point of water.